src/sass/patterns/_form-elements.scss

```diff
@@ -1,16 +1,47 @@
 // Form Elements
 // =============
 
-.vertical-separator {
-  border-right: 1px solid $color-border-separator-alt;
+.form-grid {
+  display: grid;
+  grid-template-columns: 1fr 25%;
+  flex-grow: 1;
+  width: calc(100% - 2.25rem);
+}
+
+[data-form='task-capture'] {
+  display: grid;
   height: 100%;
-  width: 1px;
 
-  .hide-separator + & {
-    display: none;
+  // using slds class since we can't access inner checkbox + label markup
+  .slds-checkbox__label {
+    &:hover {
+      .slds-form-element__label {
+        color: $brand-text-link-active;
```

Comment:
seems weird that we would have to add this hover change - do they not have one built in?

Reply:
I thought it was unusual too, but nothing is shown on the label nor checkbox (faux checkbox) and it is part of the click target: https://www.lightningdesignsystem.com/components/checkbox/

Reply:
Is there a reason we want to override the design system in this one specific case? I'd expect we either trust them on this, or override it globally?